UITestsFoundation/XCUIApplication+Assertions.swift

```diff
@@ -0,0 +1,18 @@
+import XCTest
+
+extension XCUIApplication {
+
+    public func assertLabelExists(
+        withText text: String,
+        timetout: TimeInterval = 5.0,
+        file: StaticString = #file,
+        line: UInt = #line
```

Comment:
The compiler replaces `#file` and `#line` with the path to the file and the line of code at that invoke the method. By then passing them to `XCTAssertTrue` we help Xcode reporting a possible test failure in line.

![image](https://user-images.githubusercontent.com/1218433/134476620-e2938302-a2c7-44fb-82aa-f0a093913507.png)

Reply:
> The compiler replaces #file and #line with the path to the file and the line of code at that invoke the method. By then passing them to XCTAssertTrue we help Xcode reporting a possible test failure in line.

Wow, this is mind-blowing, thank you!